3rdparty/snmalloc/allocator.cpp

```diff
@@ -20,13 +20,15 @@ void* ThreadAllocUntyped::get()
 } // namespace snmalloc
 
 #define OPEN_ENCLAVE
+#define SNMALLOC_SGX
```

Comment:
Use the SGX AAL, this should perhaps be wrapped in #ifdef OE_SGX, come to think of it.

Reply:
I have just used SNMALLOC_SGX without the OE_SGX guard for now since snmalloc is enabled only for SGX. OPTEE is single-threaded and snmalloc may not add much value there currently.